### PR TITLE
Update config and accessibility points on checklists

### LINKED ISSUE #94: Update quality assurance checklists
- [x] - Change configuration section to reflect that you should use a separate config file and version control this separately to the code, so that they can be changed independently
- [x] - Add an item for accessibility of outputs from the pipeline

### PR DESCRIPTION
**Pull request by foster999**
- Clarify that configuration files should be separate files above lower checklist
- Add output accessibility to all levels

### CODE REVIEW

**Review by joe9perry7 — APPROVED** [review]

**Review by joshlynchONS — APPROVED** [review]
Everything looks good to me.

File: book/checklist_lower.md [hunk 1]
```diff
@@ -47,11 +47,12 @@ Quality assurance checklist from [the quality assurance of code for analysis and
 ### Configuration
 
 - Credentials and other secrets are not written in code but are configured as environment variables.
-- Configuration is written as code, and is clearly separated from code used for analysis.
+- Configuration is clearly separated from code used for analysis, so that it is simple to identify and update.
 - The configuration used to generate particular outputs, releases and publications is recorded.
 
 ### Data management
 
+- Published outputs meets [accessibility regulations](https://analysisfunction.civilservice.gov.uk/area_of_work/accessibility/).
```
Comment (RuSnook) [inline book/checklist_lower.md]: verb/noun don't match: 'outputs meet' or 'output meets'

File: book/checklist_moderate.md [hunk 2]
```diff
@@ -59,12 +59,15 @@ Quality assurance checklist from [the quality assurance of code for analysis and
 ### Configuration
 
 - Credentials and other secrets are not written in code but are configured as environment variables.
-- Configuration is written as code, and is clearly separated from code used for analysis.
+- Configuration is stored in a dedicated configuration file, outside of the code.
```
Comment (RuSnook) [inline book/checklist_moderate.md]: Think this sounds nicer but what's there is fine. If you choose to update then will need updating in all instances.
```suggestion
- Configuration is stored in a dedicated configuration file, separate to the code.
```

File: book/checklist_higher.md [hunk 3]
```diff
@@ -64,12 +64,15 @@ Quality assurance checklist from [the quality assurance of code for analysis and
...
+- Configuration is stored in a dedicated configuration file, outside of the code.
+- If appropriate, multiple configuration files are used depending on system/local/user.
+- Configuration files are version controlled separately to the analysis code, so that they can be updated independently.
 - The configuration used to generate particular outputs, releases and publications is recorded.
-- If appropriate, multiple configuration files are used and interchangeable depending on system/local/user.
+- Example configuration file templates are provided alongside the code, but do not include real data.
 
 ### Data management
 
+- Published outputs meets [accessibility regulations](https://analysisfunction.civilservice.gov.uk/area_of_work/accessibility/).
```
Comment (RuSnook) [inline book/checklist_higher.md]: see comment on Lower checklist

File: book/checklist_moderate.md [hunk 4]
```diff
@@ -59,12 +59,15 @@ Quality assurance checklist from [the quality assurance of code for analysis and
...
+- If appropriate, multiple configuration files are used depending on system/local/user.
+- Configuration files are version controlled separately to the analysis code, so that they can be updated independently.
 - The configuration used to generate particular outputs, releases and publications is recorded.
-- If appropriate, multiple configuration files are used and interchangeable depending on system/local/user.
+
 
 ### Data management
 
+- Published outputs meets [accessibility regulations](https://analysisfunction.civilservice.gov.uk/area_of_work/accessibility/).
```
Comment (RuSnook) [inline book/checklist_moderate.md]: see comment on lower checklist

**Review by RuSnook — CHANGES_REQUESTED** [review]
couple of minor grammar updates and 1 suggestion

File: book/checklist_moderate.md (hunk 2 above)
Comment (foster999) [reply book/checklist_moderate.md]: Implemented this change on moderate and higher

File: book/checklist_lower.md (hunk 1 above)
Comment (foster999) [reply book/checklist_lower.md]: changed across all checklists

**Comment by foster999** [comment]
Thanks @RuSnook, I've updated those bits. Hopefully this looks alright now?

**Review by RuSnook — APPROVED** [review]
looks good

**Pull request merged**